checkers/forbidden_import_checker.py

```diff
@@ -1,26 +1,42 @@
+import astroid
 from pylint.interfaces import IAstroidChecker
 from pylint.checkers import BaseChecker
+from pylint.checkers.utils import check_messages
 
 
 class ForbiddenImportChecker(BaseChecker):
     __implements__ = IAstroidChecker
 
     name = 'custom'
     msgs = {'E9999':
-                ('You may not import any modules - you imported \033[4;34m%s\033[0m on line %s.',
```

Comment:
I understand what you did. I'm saying not to do it like that, but instead use the same message for both checks.